src/lib/modernExtend.ts

```diff
@@ -1609,12 +1609,13 @@ export function iasWarning(args?: IasWarningArgs): ModernExtend {
 
 // Uses Electrical Measurement and/or Metering, but for simplicity was put here.
 type MultiplierDivisor = {multiplier?: number; divisor?: number};
+type MultiplierDivisorMin = {multiplier?: number; divisor?: number; min?: number};
 export interface ElectricityMeterArgs {
     cluster?: 'both' | 'metering' | 'electrical';
     current?: false | MultiplierDivisor;
-    power?: false | MultiplierDivisor;
+    power?: false | MultiplierDivisorMin;
```

Comment:
```suggestion
    power?: false | (MultiplierDivisor & ReportingConfigWithoutAttribute);
```

Reply:
I can't use ReportingConfigWithoutAttribute because its elements of min/max/change aren't optional so would need to allow all three to be specified. I was trying to make this PR the lightest touch possible. This works.
```
    power?: false | (MultiplierDivisor & {min?: ReportingConfigTime});
```